README.md

```diff
@@ -32,6 +32,12 @@ and then install using this link: [backpack.tf extended sorting.user.js](https:/
 
 _____________________________________________
 
+#### 0.1.24
+
+Added:
+
+* Craft Number Sort Menu
```

Comment:
Its nitpicking, but because it isn't a menu, but a new grouping method I would just call it `Grouping by craft number`